apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generated by `createservice user.SendMessageToSupportService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/SendMessageToSupportService')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { MESSAGE_FORWARDED_TO_SUPPORT } = require('@condo/domains/notification/constants/constants')
+const { SUPPORT_EMAIL } = require('@condo/domains/common/constants/requisites')
+const { get } = require('lodash')
+const { LOCALES } = require('@condo/domains/common/constants/locale')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { v4: uuid } = require('uuid')
+const dayjs = require('dayjs')
+
+const SEND_MESSAGE_TO_SUPPORT_ATTACHMENTS_FILE_FOLDER_NAME = 'forwarded-emails-attachments'
+const fileAdapter = new FileAdapter(SEND_MESSAGE_TO_SUPPORT_ATTACHMENTS_FILE_FOLDER_NAME)
+
+const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportService', {
+    types: [
+        {
+            access: true,
+            type: `enum SendMessageToSupportLang { ${Object.keys(LOCALES).join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input SendMessageToSupportInput { dv: Int!, sender: JSON!, text: String!, email: String, attachments: [Upload], os: String!, appVersion: String!, lang: SendMessageToSupportLang!, meta: JSON! }',
+        },
+        {
+            access: true,
+            type: 'type SendMessageToSupportOutput { id: String!, status: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canSendMessageToSupport,
+            schema: 'sendMessageToSupport(data: SendMessageToSupportInput!): SendMessageToSupportOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data } = args
+                const { dv, sender, text, email: emailFrom, attachments = [], os, appVersion, lang, meta } = data
```

Comment:
you need to validate the emailFrom field here.

Reply:
Done! :)